src/camera_streamer/setup.py

```diff
@@ -0,0 +1,28 @@
+from setuptools import find_packages, setup
+
+package_name = 'camera_streamer'
+
+setup(
+    name=package_name,
+    version='0.0.0',
+    packages=find_packages(exclude=['test']),
+    data_files=[
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
+    ],
+    install_requires=['setuptools'],
+    zip_safe=True,
+    maintainer='pratik',
+    maintainer_email='pratikphadte19@gmail.com',
+    description='TODO: Package description',
```

Comment:
I won't stop you from merging because of this, but if it's not too much trouble you should change this before merging.